mm2src/coins/hd_wallet_storage/sqlite_storage.rs

```diff
@@ -1,9 +1,11 @@
+#![allow(deprecated)] // TODO: remove this once rusqlite is >= 0.29
```

Comment:
What is preventing us from updating to  `rusqlite >= 0.29` in this PR? Just curious since I am sure you have a valid reason :)

Reply:
`0.29` removes the deprecated functions, which requires a lot more changes on rusqlite related implementations.